plugin/pkg/admission/namespace/lifecycle/admission.go

```diff
@@ -45,8 +45,9 @@ const (
 	missingNamespaceWait = 50 * time.Millisecond
 )
 
-func init() {
-	admission.RegisterPlugin(PluginName, func(io.Reader) (admission.Interface, error) {
+// Register registers a plugin
+func Register(plugins *admission.Plugins) {
+	plugins.Register(PluginName, func(io.Reader) (admission.Interface, error) {
```

Comment:
yay, we'll get rid of an init. This was being done upstream and we're following? If so, that's very good and makes me happy.